test/testutil/genesis.go

```diff
@@ -0,0 +1,300 @@
+package testutil
+
```

Comment:
this file can be deleted, and will be in my PR so up to you if you want to remove it now or later

my apologies for merging it in, it was appendix code from something I had been working on